planetarium/NineChronicles
# Cache Raid:Season reward, Add notifying as Mail

### Description

1. Cache season reward of Raid(World boss).
2. Notify season reward as Mail(Just in client, not affect to chain or store).
3. If local has old-cached data, delete that.

### How to test

1. Set `CommandLineOptions.onBoardingHost`.
2. Check mailbox of avatar that recieved season reward.
4. Change avatar, or anything... test mailbox and notification.

### Related Links

[시즌 보상 캐싱, 메일로 보상 안내](https://app.asana.com/0/0/1203138597935695/f)

### Required Reviewers

@planetarium/9c-dev 

### Screenshot

![image](https://user-images.githubusercontent.com/48484989/196610961-02634f16-cf32-4b86-87ee-aa5669a9cc4e.png)
![image](https://user-images.githubusercontent.com/48484989/196612902-d2a18563-96dd-4e83-ad47-c89352088fab.png)
